test/conformance/chainsaw/validate/clusterpolicy/standard/apicalls/default/ns.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
why are there 2 namespaces

Reply:
One for exceptyion and another for staging-ns